app/models/apps/parliament_vote_app/application_form.rb

```diff
@@ -74,7 +74,7 @@ def year
       end
 
       def same_delivery_address?
-        same_delivery_address == '1'
+        same_delivery_address == '1' || same_delivery_address.blank?
```

Comment:
Ked submitnem form bez zvolania delivery, tak to vyhodilo error pre obidva vetvy :

<img width="789" alt="Screen Shot 2019-11-11 at 12 41 32" src="https://user-images.githubusercontent.com/1857751/68619844-14887180-0481-11ea-873b-f791d31a1bd7.png">

Reply:
Nerozumiem.

Reply:
Hmm no `same_delivery_address?` predpoklada ze sme vybrali jednu z moznosti.
Je to v podstate len na nas ktoru moznost berieme ako default ked nic neni vybrane, a myslim ze prva moznost dava vacsi zmysel.

Inac nastane toto:

<img width="793" alt="Screen Shot 2019-11-11 at 13 40 54" src="https://user-images.githubusercontent.com/1857751/68623251-ead34880-0488-11ea-92a3-2f73c9793076.png">

Reply:
Aha tak to potom dava vacsi zmysel otocit na `custom_delivery_address?` a nastavit tie validacie zvysne s kontextom `if: ->(f) { f.custom_delivery_address?`